Composer/packages/ui-plugins/select-skill-dialog/src/SkillEndpointField.tsx

```diff
@@ -26,8 +26,10 @@ export const SkillEndpointField: React.FC<FieldProps> = (props) => {
   const deserializedValue = typeof uiOptions?.serializer?.get === 'function' ? uiOptions.serializer.get(value) : value;
 
   const handleChange = (newValue: any) => {
-    const serializedValue =
-      typeof uiOptions?.serializer?.set === 'function' ? uiOptions.serializer.set(newValue) : newValue;
+    const serializedValue = newValue;
+    if (typeof uiOptions?.serializer?.set === 'function') {
+      uiOptions.serializer.set(newValue);
```

Comment:
I'm confused about what the intent is here. `uiOptions.serializer.set` returns a value; however, that value isn't being used. Do we still need the serializers?

Reply:
 `=`setting.skill\['skillName'\].endpointUrl\` is in the schema. It gets transformed to the endpointName through get fn

Reply:
 `=`setting.skill\['skillName'\].endpointUrl\` is in the schema. It gets transformed to the endpointName through get fn